config/locales/en.yml

```diff
@@ -153,6 +153,13 @@ en:
     company: 'company'
     all_users: 'All Users'
     no_user_to_show: 'No User to show...'
+    user_created_successfully: 'User created successfully'
```

Comment:
Can we not use the generic key defined on line 249?

Reply:
done